src/components/resources/Observation/ObservationGraph.js

```diff
@@ -1,9 +1,12 @@
 import React from 'react';
 import _get from 'lodash/get';
-import _has from 'lodash/has';
+import _isUndefined from 'lodash/isUndefined';
 
 const ObservationGraph = props => {
-  if (_has(props, 'referenceRange') && _has(props, 'valueQuantity.value')) {
+  if (
+    !_isUndefined(props.referenceRange) &&
+    !_isUndefined(_get(props, 'valueQuantity.value'))
```

Comment:
Perhaps this would be a simpler way to check?

```suggestion
    props.referenceRange && _isNumber(_get(props, 'valueQuantity.value'))
```